legend-engine-language-pure-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/core/M3LexerGrammar.g4

```diff
@@ -10,7 +10,7 @@ LET:                                        'let';
 ALL_VERSIONS:                               'allVersions';
 ALL_VERSIONS_IN_RANGE:                      'allVersionsInRange';
 
-BYTE_STREAM_FUNCTION:                       'byteStream';
+BYTE_ARRAY_FUNCTION:                        'byteArray';
```

Comment:
I think it would be better to call this "toBytes" or something like that. Calling it "byteArray" seems to imply that Byte[*] is implemented as an array, which is not always going to be the case.
```suggestion
BYTE_ARRAY_FUNCTION:                        'toBytes';
```

Reply:
done